.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["react-app", "eslint-config-triggr"],
+  "rules": {
+    "no-console": "off",
+    // This doesn't work because the react-scripts webpack ignores this file. Left here as a
```

Comment:
I'd use the eslintrc from triggr_api (except for the rules marked as TODO) instead of a React one.

Eslint is very useful for things like unused variables, which are usually bugs.

Reply:
I ended up copy/pasted our eslint-config-triggr rules, I'm worried we won't be able to pull the private repo from a public one. It was also preventing me from running the lint locally.